File: Turculescu_Samuel/Week_5/Domain/Entities/OrderItem.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Persistence.EfCore
{
    public partial class OrderItem
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int MenuItemId { get; set; }
        public int OrderId { get; set; }
        public uint Quantity { get; set; }
        public string SpecialRequests { get; set; }      
        public double Price { get; set; }

        public virtual MenuItem MenuItem { get; set; }
        public virtual Order Order { get; set; }
    }
}

File: Turculescu_Samuel/Week_8/order-and-pay-employee/src/models/orderItem.ts
import { MenuItem } from "./menuItem";
import { Order } from "./order";

export type OrderItem = {
    id: string,
    name: string,
    menuItemId: string,
    orderId: string,
    quantity: number,
    specialRequests: number,
    price: number    
    
    menuItem: MenuItem,
    order: Order
}